wagtail/snippets/urls.py

```diff
@@ -7,7 +7,6 @@
 urlpatterns = [
     path('', snippets.index, name='index'),
 
-    path('choose/', chooser.choose, name='choose_generic'),
```

Comment:
To clarify, choose_generic was never a usable URL route in its own right - it only served to generate the partial URL to pass to JS (where the model parameter would then get tacked on).